content/docs/instrumenting/exporters.md

```diff
@@ -98,6 +98,7 @@ separate exporters are needed:
    * [Etcd](https://github.com/coreos/etcd)
    * [Kubernetes-Mesos](https://github.com/mesosphere/kubernetes-mesos)
    * [Kubernetes](https://github.com/GoogleCloudPlatform/kubernetes)
+   * [linkerd](https://github.com/BuoyantIO/linkerd)
```

Comment:
We usually lower-case it regardless of whether it's at the beginning of a sentence or a list, but I'm honestly not too concerned about this. Will uppercase if you want.